contrib/guix/libexec/build.sh

```diff
@@ -229,7 +230,7 @@ mkdir -p "$OUTDIR"
 ###########################
 
 # CONFIGFLAGS
-CONFIGFLAGS="--enable-reduce-exports --disable-bench --disable-gui-tests --disable-fuzz-binary"
+CONFIGFLAGS+=" --enable-reduce-exports --disable-bench --disable-gui-tests --disable-fuzz-binary"
```

Comment:
IMO it seems strange to have for deterministic builds some custom specified configure params. That's not exactly what deterministic builds are made for....

it doesn't look for me as something that supposed to be widely or often be used; if you need to update CONFIGFLAGS for a specific experimental local build to let this option included in this config permanently  you may change CONFIG_FLAGS here as well for once.

Could you give me an example how are you going to use this external param?

Reply:
Sorry, forgot to update description. Fixed.